admin_guide/install/deploy_router.adoc

```diff
@@ -300,24 +300,48 @@ The HAProxy router is based on a
 link:http://golang.org/pkg/text/template/[*golang* template] that
 generates the HAProxy configuration file from a list of routes. If you
 want a customized template router to meet your needs, you can customize
-the template file, build a new docker image, and run a customized router.
+the template file, build a new Docker image, and run a customized router.
 
 One common case for this might be implementing new features within the
 application back ends. For example, it might be desirable in a highly-available
-setup to use stick-tables that synchronizes between peers. The router plug-in
-provides all the facilities necessary to make this customization.
+setup to link:#using-stick-tables[use stick-tables] that synchronizes between
+peers. The router plug-in provides all the facilities necessary to make this
+customization.
 
-.Using Stick Tables
+You can obtain a new *_haproxy-config.template_* file from the latest router
+image by running:
+
+----
+ifdef::openshift-enterprise[]
+# docker run --rm --interactive=true --tty --entrypoint=cat \
+    registry.access.redhat.com/openshift3/ose-haproxy-router:v3.0.1.0 haproxy-config.template
+endif::[]
+ifdef::openshift-origin[]
+# docker run --rm --interactive=true --tty --entrypoint=cat \
+    registry.access.redhat.com/openshift3/origin-haproxy-router haproxy-config.template
```

Comment:
Origin should just be
`openshift/origin-haproxy-router  haproxy-config.template`